tutorials/self-signed-docker-registry-on-gke.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Using a self-signed Docker Registry on GKE
+description: Using a self-signed Docker Registry on GKE.
+author: samos123
+tags: Kubernetes Engine, GKE, Docker Registry
+date_published: 2020-01-28
+---
+
+This tutorial shows how to use a Docker registry that was signed by a 
+non-trusted CA with Google Kubernetes Engine (GKE). Let's assume your company
+has a private CA that's not trusted by default.
+All internal services such as on-prem docker registry have their certs signed
+by this private CA. This would cause GKE to prevent pulling from the on-prem
```

Comment:
I would show the error by doing this from a dockerhub registry or some other non trusted registry